From aa7af9f4e7ca4bcb620335f3f6c86e6dbafc1807 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Fri, 29 Apr 2016 18:17:16 -0400
Subject: [PATCH 0897/1936] CMake: Remove LibloEnable Flag

This flag should have always been enabled since the start
of the 2.5.x series.
---
 src/CMakeLists.txt    | 20 +++++++-------------
 src/UI/CMakeLists.txt |  4 +---
 2 files changed, 8 insertions(+), 16 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 365c129c..f3ae89f6 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -78,8 +78,6 @@ SET (LashEnable ${LASH_FOUND} CACHE BOOL
     "Enable LASH Audio Session Handler")
 SET (DssiEnable ${DSSI_FOUND} CACHE BOOL
     "Enable DSSI Plugin compilation")
-SET (LibloEnable ${LIBLO_FOUND} CACHE BOOL
-    "Enable Liblo")
 SET (NoNeonPlease False CACHE BOOL
     "Workaround For Broken Neon Detection")
 SET (PluginLibDir "lib" CACHE STRING
@@ -228,13 +226,12 @@ if(LashEnable)
 	list(APPEND AUDIO_LIBRARY_DIRS ${LASH_LIBRARY_DIRS})
 	message(STATUS "Compiling with lash")
 endif()
-if(LibloEnable)
-	include_directories(${LIBLO_INCLUDE_DIRS})
-	add_definitions(-DUSE_NSM=1)
-    list(APPEND AUDIO_LIBRARIES ${LIBLO_LIBRARIES})
-	list(APPEND AUDIO_LIBRARY_DIRS ${LIBLO_LIBRARY_DIRS})
-	message(STATUS "Compiling with liblo")
-endif()
+
+include_directories(${LIBLO_INCLUDE_DIRS})
+add_definitions(-DUSE_NSM=1)
+list(APPEND AUDIO_LIBRARIES ${LIBLO_LIBRARIES})
+list(APPEND AUDIO_LIBRARY_DIRS ${LIBLO_LIBRARY_DIRS})
+message(STATUS "Compiling with liblo")
 
 # other include directories
 include_directories(${ZLIB_INCLUDE_DIRS} ${MXML_INCLUDE_DIRS})
@@ -356,9 +353,7 @@ if(NtkGui)
 endif()
 
 if(NOT FltkGui AND NOT NtkGui)
-    if(LibloEnable)
-        set(NSM_WORKAROUND UI/NSM.C UI/NSM/Client.C)
-    endif()
+    set(NSM_WORKAROUND UI/NSM.C UI/NSM/Client.C)
     add_library(zynaddsubfx_gui_bridge STATIC UI/ConnectionDummy.cpp ${NSM_WORKAROUND})
     add_definitions(-DNO_UI=1)
 endif()
@@ -502,7 +497,6 @@ package_status(X11_FOUND        "x11      " "found"   ${Yellow})
 package_status(CXXTEST_FOUND    "CxxTest  " "found"   ${Yellow})
 package_status(LashEnable       "Lash     " "enabled" ${Yellow})
 package_status(DssiEnable       "DSSI     " "enabled" ${Yellow})
-package_status(LibloEnable      "liblo    " "enabled" ${Red})
 package_status(CompileTests     "tests    " "enabled" ${Yellow})
 package_status(AlsaEnable       "ALSA     " "enabled" ${Yellow})
 package_status(JackEnable       "JACK     " "enabled" ${Yellow})
diff --git a/src/UI/CMakeLists.txt b/src/UI/CMakeLists.txt
index 5ab0cd64..bca6a23a 100644
--- a/src/UI/CMakeLists.txt
+++ b/src/UI/CMakeLists.txt
@@ -22,9 +22,7 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR})
 set_source_files_properties(UI/MasterUI.h PROPERTIES GENERATED 1)
 fltk_wrap_ui(zynaddsubfx_gui ${UI_fl_files})
 
-if(LibloEnable)
-    set(zynaddsubfx_gui_FLTK_UI_SRCS ${zynaddsubfx_gui_FLTK_UI_SRCS} NSM.C NSM/Client.C)
-endif()
+set(zynaddsubfx_gui_FLTK_UI_SRCS ${zynaddsubfx_gui_FLTK_UI_SRCS} NSM.C NSM/Client.C)
 if(NtkGui)
     add_definitions(-DPIXMAP_PATH="${CMAKE_INSTALL_PREFIX}/share/zynaddsubfx/pixmaps/")
     add_definitions(-DSOURCE_DIR="${CMAKE_SOURCE_DIR}")
-- 
2.47.0

